case_studies/weather_dependent_app_example/README.md

```diff
@@ -0,0 +1,36 @@
+# Overview
```

Comment:
I'd change the app name to something more specific. Maybe "Dependency checking setup page app".

Reply:
Updated.